calculator_test.go

```diff
@@ -2,23 +2,288 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+	tests := []struct {
+		in1, in2, want float64
```

Comment:
It's a curious fact about variable names that our brains aren't very good at distinguishing words with similar beginnings. To make it easier to read this, try something like `a, b` instead.